.gitignore

```diff
@@ -0,0 +1,31 @@
+HELP.md
+target/
+!.mvn/wrapper/maven-wrapper.jar
+!**/src/main/**
+!**/src/test/**
+
+### STS ###
+.apt_generated
+.classpath
```

Comment:
Having `.classpath`, `.project`, and (though this is less important) `.settings` in `.gitignore` means anyone checking this repo out from GitHub can't just open it in Eclipse, they have to use the "import from Maven" feature. So you may want to remove those two (or three) lines. But it's up to you.

Reply:
Yeah, sound good.